openmetadata-service/src/main/java/org/openmetadata/service/OpenMetadataApplication.java

```diff
@@ -264,16 +264,23 @@ public void run(OpenMetadataApplicationConfig catalogConfig, Environment environ
     // Asset Servlet Registration
     registerAssetServlet(catalogConfig.getWebConfiguration(), environment);
 
-    // Handle Pipeline Service Client Status job
-    PipelineServiceStatusJobHandler pipelineServiceStatusJobHandler =
-        PipelineServiceStatusJobHandler.create(
-            catalogConfig.getPipelineServiceClientConfiguration(), catalogConfig.getClusterName());
-    pipelineServiceStatusJobHandler.addPipelineServiceStatusJob();
+    // Handle Services Jobs
+    registerHealthCheckJobs(catalogConfig);
 
     // Register Auth Handlers
     registerAuthServlets(catalogConfig, environment);
   }
 
+  private void registerHealthCheckJobs(OpenMetadataApplicationConfig catalogConfig) {
+    ServicesStatusJobHandler pipelineServiceStatusJobHandler =
```

Comment:
lets name it as healthCheckStatusHandler. Pipeline is used in different context causes confusion

Reply:
done